README.md

```diff
@@ -28,6 +28,8 @@ The plugin makes available to the job the following parameter variables:
 - `${sourceCommitHash}`
 - `${destinationCommitHash}`
 
+Of those variables, `${destinationRepositoryOwner}` and `${destinationRepositoryName}` are available in the project configuration.
```

Comment:
Can you add the example from the commit message here too? I feel without it users might have trouble understanding what it means (if I reference `sourceCommitHash` from e.g. a shell step, that is also a kind of project configuration).

Reply:
Thank you, I'm adding some clarifications.